src/containers/InputReservationContainer.jsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import { useSelector, useDispatch } from 'react-redux';
+
+import InputReservation from '../components/InputReservation';
+import {
+  updateReservation,
+  addReservation,
+} from '../modules/actions';
+
+export default function InputReservationContainer() {
+  const { reservation } = useSelector((state) => ({
+    reservation: state.reservation,
+  }));
+
+  const dispatch = useDispatch();
+
+  function handleChange({ event, target }) {
+    const { value } = event.target;
+    const eventTargets = {
+      name: updateReservation({ target, value }),
+      category: updateReservation({ target, value }),
+      address: updateReservation({ target, value }),
+    };
+
+    dispatch(eventTargets[target]);
+  }
```

Comment:
`Object Literal Lookup`을 사용해서 처리해보신것 같아요~ 예외처리가 필요한 경우에는 매우 유용할 수 있지만, 이 경우에는 바로 ditpatch에서 action을 호출하는게 좋아보여요.

```suggestion
  function handleChange({ event, target }) {
    const { value } = event.target;
    dispatch(updateReservation({target, value}));
  }
```

Reply:
그러고보니 if문이 필요 없이 그냥 넣으면 되는 부분이었군요 ㅠㅠ